### DISCUSSION TITLE
Antenna Accuracy for a LEO satellite

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#109] [Q&A] [answered]
Hello 

Along the question that Parisa asked in the discussion topic 106 one other question came up with regards to antenna accuracy pattern values in the GnssAntennaDefinitonCreate program (Sentinel 1A). We do not understand where the values come from, more specifically: offsetx, offset y, offset z delta Azimuth, deltaZenith, maxZenith and values. We could not find those in the Sentinel handbooks and they are not reflected in the 3 files ((sat)Temporal.txt, (sat)CenteOfGravity.txt and sat _antennas.txt <-> (sat)gnssTypes.txt not part of config directory)  that are located in the Sentienl1A config directory, 

If we understand the source of those values and /or any input files to get those values then we can create the same information for another Leo satellite.

Thank you for you continued help and feedback

Cheers
Jean
PS - Is there a plan to do some short training on GROOPS (like a 2 -3 day) online course. (in addition to the videos which were very helpful for getting started)? 
![Cap1](https://user-images.githubusercontent.com/100558965/191773458-4e9bcd14-c422-48c9-852d-5fb1716b6fa0.png)

**Comment by tmayerguerr** [comment, marked as answer]
Hi Jean and Parisa,

Normally, an elevation-dependent weighting is introduced for processing. In GROOPS the weight p=1/sigma is defined by the accuracy sigma. GROOPS uses the AntennaDefinition file format to allow flexible azimuth and elevation dependent weighting. The offsets in the file are not used and are set to zero.

Your example shows a generic cos-weighting with 7 cm and 1 mm accuracy in zenith direction for code and phase, respectively. The expression is evaluated every 5 degrees in zenith direction and only one value (for all 360 degrees) in azimuth direction. The factor 0.96 is used to get a finite value at the horizon. The values are only educated guesses.

A more sophisticated azimuth and elevation dependent weighting is based on the real accuracies estimated from the residuals. The residuals are computed in an intermediate processing step. The computation is done with [GnssResiduals2AccuracyDefinition](https://groops-devs.github.io/groops/html/GnssResiduals2AccuracyDefinition.html).

Online course: There is no course scheduled at the moment. We are very busy with our daily work right now: projects, conferences and teaching.

regards
Torsten

**Comment by Bemtgenj** [comment]
HI Torsten

Thanks much for the quick reply. It is clearer now where the values come from.

Cheers
Jean
